src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,26 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Sorts all person in the address book according to name
+ */
+
```

Comment:
Extra line. Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.